From c23c048eaf5e882a23c72e8bd0fab15f12a58425 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Fri, 14 Nov 2008 14:58:46 -0800
Subject: Staging: comedi: fix build error in comedilib.h

If anyone included this file, it would break the build.
This patch fixes the issue.

Cc: David Schleef <ds@schleef.org>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/comedilib.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/comedi/comedilib.h
+++ b/drivers/staging/comedi/comedilib.h
@@ -24,7 +24,7 @@
 #ifndef _LINUX_COMEDILIB_H
 #define _LINUX_COMEDILIB_H
 
-#include <linux/comedi.h>
+#include "comedi.h"
 
 /* Kernel internal stuff.  Needed by real-time modules and such. */
 
